b_cfn_custom_userpool_authorizer/source/policy_document.py

```diff
@@ -31,5 +34,18 @@ def create_policy_statement(
                         'Effect': 'Allow' if allow else 'Deny'
                     }
                 ]
+            },
+            'context': {
```

Comment:
We are exposing here quite a lot of data. Is this a standard practice?

Reply:
The same claims are exposed from the standard Cognito authorizer (CfnAuthorizer). Also, [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-mapping-template-reference.html#context-variable-reference) does suggest that these context variables, among many others, could be used for authorizers.